documentation/jetty/modules/programming-guide/pages/arch/threads.adoc

```diff
@@ -144,25 +151,39 @@ hide time-axis
 
 ----
 
-This mode may operate like <<execution-strategy-pc,`Produce-Consume`>> when the take over production task run, for example, by thread `Thread 3` takes time to be executed (for example, in a busy server): then thread `Thread 2` will produce one task and run it, then produce another task and run it, etc. -- `Thread 2` behaves exactly like the `Produce-Consume` mode.
+This mode may operate like <<execution-strategy-pc,`Produce-Consume`>> when the take-over production task run, for example, by thread `Thread 3` takes time to be executed (for example, in a busy server): then thread `Thread 2` will produce one task and run it, then produce another task and run it, etc. -- `Thread 2` behaves exactly like the `Produce-Consume` mode.
```

Comment:
The definition of "take-over production task" is unclear.
Is is mentioned elsewhere in a previous commit?